src/main/resources/com/cloudbees/jenkins/plugins/sshagent/SSHAgentStep/help-credentials.html

```diff
@@ -0,0 +1,3 @@
+<div>
+    Select credentials from the list of credentials available to the job.
```

Comment:
Possible alternate phrasing with two examples included inside the help?
```suggestion
    List of credentials to be used by the sshagent step.
    The Pipeline Syntax Snippet Generator will guide the user to select a single credential from the list of available, like this:
    <code>
    sshagent(credentials: ['my-credential-id']) {
        // some block
    }
    </code>
    The step also accepts a list of credential, like this:
    <code>
    sshagent(credentials: ['my-credential-id', 'my-alternate-credential', 'another-credential']) {
        // some block
    }
    </code>
```

Reply:
Thank you @MarkEWaite